src/views/InvolvementProfile/components/Membership/components/AdminCard/index.js

```diff
@@ -18,6 +18,7 @@ import GordonDialogBox from 'components/GordonDialogBox';
 import { useParams } from 'react-router';
 import involvementService from 'services/activity';
 import membershipService from 'services/membership';
+import utils from 'services/utils';
```

Comment:
Utils does not have a default export, you'll want to `import { stripDomain } from 'services/utils'`